mk/config.mk

```diff
@@ -1035,7 +1035,9 @@ endif
 CFG_CALLOUT ?= $(CFG_CORE_ASYNC_NOTIF)
 
 # Enable notification based test watchdog
-CFG_NOTIF_TEST_WD ?= $(CFG_ENABLE_EMBEDDED_TESTS)
+ifeq (yyy,$(CFG_ENABLE_EMBEDDED_TESTS)$(CFG_CALLOUT)$(CFG_CORE_ASYNC_NOTIF))
```

Comment:
as a sanity test on the config values, prefer
`ifeq (y-y-y,$(CFG_ENABLE_EMBEDDED_TESTS)-$(CFG_CALLOUT)-$(CFG_CORE_ASYNC_NOTIF))`

That said, I'd rather suggest:
```makefile
CFG_NOTIF_TEST_WD ?= $(call cfg-all-enabled,CFG_ENABLE_EMBEDDED_TESTS CFG_CALLOUT CFG_CORE_ASYNC_NOTIF)